edge/pkg/edged/edged.go

```diff
@@ -340,6 +345,7 @@ func getConfig() *Config {
 	conf.version = config.CONFIG.GetConfigurationByKey("edged.version").(string)
 	conf.DockerAddress = config.CONFIG.GetConfigurationByKey("edged.docker-address").(string)
 	conf.runtimeType = config.CONFIG.GetConfigurationByKey("edged.runtime-type").(string)
+	conf.cgroupDriver = config.CONFIG.GetConfigurationByKey("edged.cgroup-driver").(string)
```

Comment:
Any updates? And we have already added the `cgroup-driver` tag in v1.1.

Reply:
Sorry for not replying because of some trouble. The `cgroup-driver` is actually added in v1.1, may be it has been fixed in v1.1 and master branch. 

I have some problems in deployment in master branch, edge node `NotReady` in cloud, and I cannot test whether the crio works ok in master branch.